docs/howto/assets/eks/files/bootstrap.sh

```diff
@@ -7,6 +7,35 @@ set -o pipefail
 set -o nounset
 set -o errexit
 
+function print_help {
+    echo "usage: $0 [options] <cluster-name>"
+    echo "Bootstraps a Krustlet instance into an EKS cluster"
+    echo ""
+    echo "-h,--help print this help"
+    echo "--krustlet-node-labels Add extra labels to Krustlet."
+}
+
+while [[ $# -gt 0 ]]; do
+    key="$1"
+    case $key in
+        -h|--help)
+            print_help
+            exit 1
+            ;;
+        --krustlet-node-labels)
```

Comment:
isn't the flag name `--node-labels`?

Reply:
This is the bootstrap script that sets up certs, env vars etc. Do we want this flag to name as `--node-labels` too? The `--krustlet` prefix here is to make it clear that this is krustlet's node labels. This script may set up things other than krustlet in the future. I don't have strong opinions on removing the `--krustlet` prefix for the bootstrap script as well.

Reply:
You can see the EKS bootstrap script as an example which also has the kubelet prefix: https://github.com/awslabs/amazon-eks-ami/blob/master/files/bootstrap.sh#L22

Reply:
Personally, I think `--krustlet-node-labels` is fine here.

Reply:
ah thank you! I just noticed this was translated over to NODE_LABELS afterwards.

One question with that behaviour though: wouldn't that override what I set locally for `NODE_LABELS`?

e.g. if I do

```
$ export NODE_LABELS="foo=bar"
./bootstrap.sh
```

As far as I can tell, no node labels will be applied as the script will read from `--krustlet-node-labels` and override the existing environment variable.